os/net/lwip/sys/arch/sys_arch.c

```diff
@@ -483,21 +482,22 @@ u32_t sys_arch_sem_wait(sys_sem_t *sem, u32_t timeout)
 			 * awakened by the receipt of a signal.
 			 * Restart If signal is EINTR else break if ETIMEDOUT
 			 */
-			if (status == -ECANCELED) {
-				return SYS_ARCH_CANCELED;
-			} else if (status == -ETIMEDOUT) {
+			int err_no = get_errno();
+			if (err_no == ETIMEDOUT) {
 				return SYS_ARCH_TIMEOUT;
-			} else {
+			} else if (err_no == ENOMEM) {
+				LWIP_ASSERT("status == ENOMEM", 0);
+			} else if (err_no == ECANCELED) {
+				return SYS_ARCH_CANCELED;
+			}else {
```

Comment:
} else {
Could you add a space between `} else`?

Reply:
Thanks I'll add a space